src/utils/Logger.js

```diff
@@ -40,7 +38,8 @@ function isNonEmptyString(value :any) {
 
 function getMessagePrefix(loggerLevel :string, loggerName :string) {
 
-  return `[${format(new Date(), TIMESTAMP_FORMAT)} ${loggerLevel.toUpperCase()} ${__PACKAGE__}] ${loggerName}`;
+  // eslint-disable-next-line max-len
+  return `[${DateTime.local().toSQL({ includeOffset: false })} ${loggerLevel.toUpperCase()} ${__PACKAGE__}] ${loggerName}`;
```

Comment:
In all other `Logger` files, I got rid of any date/format dependencies, and just changed it to this:

```
return `[${(new Date()).toISOString()} ${loggerLevel.toUpperCase()} ${__PACKAGE__}] ${loggerName}`;
```

You can just copy it directly: https://github.com/openlattice/lattice-js/blob/develop/src/utils/Logger.js